src/main/java/frc/robot/subsystems/Input.java

```diff
@@ -10,19 +10,23 @@
 
 // Input Class For Joystick/Controller Input Functions
 public class Input {
+    private Input(){
+        throw new IllegalStateException("Input Class");
+      }
 
-    private static Joystick rJoystick = new Joystick (JoystickConstants.rotJoystickPort);
-    private static Joystick tJoystick = new Joystick (JoystickConstants.transJoystickPort);
+    private static Joystick rJoystick = new Joystick (JoystickConstants.ROT_JOYSTICK_PORT);
+    private static Joystick tJoystick = new Joystick (JoystickConstants.TRANS_JOY_PORT);
     
-    private static XboxController xboxController = new XboxController(JoystickConstants.xboxControllerPort);
+    private static XboxController xboxController = new XboxController(JoystickConstants.XBOX_PORT);
 
-    public static int DPADUP = 0;
-    public static int DPADRIGHT = 90;
-    public static int DPADDOWN = 180;
-    public static int DPADLEFT = 270;
+    public static final int DPADUP = 0;
+    public static final int DPADRIGHT = 90;
+    public static final int DPADDOWN = 180;
+    public static final int DPADLEFT = 270;
 
 
     public static boolean resetGyro(){return rJoystick.getRawButton(3);}
```

Comment:
Use named constants instead of magic numbers.

Reply:
Applies to all other magic numbers in this file.

Reply:
The Input file is like a constants in the sense that it does nothing on its own and is just used for other classes to use controller input. Therefore the magic numbers are simply just joystick ports as suggested by the method names.